source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+The `QIIME 2 forum <https://forum.qiime2.org/>`__ contains a wealth of information on how to perform microbiome data processing and analysis, as well as fruitful discussion on best practices in this space.
+
+QIIME 2 also encourages making **microbiome research more reproducible**.
+QIIME 2 reduces inappropriate analyses by defining specific data types and restricting methods only to their appropriate data type inputs.
+It also keeps track of everything that's been done to a given data file in the :doc:`provenance <../concepts>` metadata associated with each QIIME 2 artifact.
+Furthermore, by wrapping tools into a common framework, **data processing pipelines are streamlined**: with QIIME 2, you'll be able to re-write what was once a collection of scripts in different coding languages, calling different executables and re-formatting data inputs and outputs in most intermediate steps, into one (or a few) simple bash scripts.
+
+Finally, QIIME 2 is **open-sourced and specifically designed for experienced researchers to contribute and expand the reach of their work.**
+QIIME 2 is really just a collection of wrappers called :doc:`plugins <../plugins/index>`, which can be written for any software, package, or other installable executables.
+Writing a QIIME 2 plugin for a method that you develop instantly makes it accessible and usable by thousands of users.
+
+Pro-tips for power users
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+That said, here are a few tips we've learned that should substantially improve your experience in transitioning your workflows to QIIME 2:
+
+**Pro-tip #1: QIIME 2 artifacts are just zip files**.
+If at any point you want to look at what actual files are in the ``.qza`` artifact, you can use :doc:`qiime tools export <exporting>` to extract the data file directly (which is basically just a wrapper for ``unzip``).
+Alternatively, you can also unzip your artifact directly (``unzip -k file.qza``) and look through the files in the ``data/`` folder.
+
+**Pro-tip #2: the QIIME 2 command line interface tools are slow because they have to unzip and re-zip the data contained in the artifacts each time you call them.**
+If you need to process your data more interactively, you might want to use the Python API - it is much faster since objects can be simply stored in memory.
+You can learn more about the different `QIIME 2 interfaces <https://docs.qiime2.org/2018.6/interfaces/>`__.
+
+**Pro-tip #3: to enable tab-complete in QIIME 2**, run ``source tab-qiime``.
+
+Data processing steps
+---------------------
+
+The processing steps we'll cover in this tutorial include:
+
+1. Importing data into QIIME 2
+2. Demultiplexing data (i.e. mapping each sequence to the sample it came from)
+3. Removing non-biological parts of the sequences (i.e. primers)
+4. Performing quality control and:
+
+   -  denoising sequences with DADA2 or deblur, and/or
+   -  quality filtering, length trimming, and clustering with VSEARCH or dbOTU
+
+5. Assigning taxonomy
+
```

Comment:
maybe mention at the end: "See the ``:doc:`overview tutorial <blah>` `` and ``:doc:`available plugins <blah>` `` for additional optional downstream analysis steps."

You cover the important ones here, but just want to make sure users reading this don't get the impression that that's all.